wrangler-core/src/test/java/io/cdap/directives/column/CopyTest.java

```diff
@@ -96,4 +99,33 @@ public void testForceCopy() throws Exception {
     Assert.assertEquals(rows.get(2).getValue("body_2"), rows.get(2).getValue("body_1"));
   }
 
+  @Test
+  public void testGetOutputSchemaForCopiedColumn() throws Exception {
+    String[] directives = new String[] {
+      "copy :col_B :col_A true",
```

Comment:
can we also add a test where the destination column is not present in input schema